src/db/z_db.erl

```diff
@@ -371,7 +383,7 @@ insert(Table, Context) ->
 
 %% @doc Insert a row, setting the fields to the props.  Unknown columns are serialized in the props column.
 %% When the table has an 'id' column then the new id is returned.
-%% @spec insert(Table::atom(), Props::proplist(), Context) -> {ok, Id} | Error
+-spec insert(table_name(), props(), #context{}) -> boolean().
```

Comment:
The spec in line 386 uses a record, please define a type for the record and use that instead.